### DISCUSSION TITLE
Final version of V3 development

### DISCUSSION THREAD

**Discussion by wandell** [RenderToolbox/pbrt2ISET#49] [Announcements]
At this time, we are moving all of our new developments to ISET3d-V4.  That repository may become the new 'ISET3d', and this one is likely to be renamed ISET3d-V3. 

We will probably be copying the relevant portions of the wiki pages over in some way, as well.

Thank you PBRT V3.  On to PBRT V4!

<hr /><em>This discussion was created from the release <a href='https://github.com/ISET/iset3d/releases/tag/v3-final'>Final version of V3 development</a>.</em>

**Comment by digital-pro** [comment]
Because I’m not-smart-enough to know better, do you want me to start trying to move wiki pages to v4?

 

 

From: Brian Wandell ***@***.***> 
Sent: Wednesday, January 19, 2022 3:40 PM
To: ISET/iset3d ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ISET/iset3d] Final version of V3 development (Discussion #49)

 

At this time, we are moving all of our new developments to ISET3d-V4. That repository may become the new 'ISET3d', and this one is likely to be renamed ISET3d-V3.

We will probably be copying the relevant portions of the wiki pages over in some way, as well.

Thank you PBRT V3. On to PBRT V4!

  _____  

This discussion was created from the release Final version of V3 development <https://github.com/ISET/iset3d/releases/tag/v3-final> . 

—
Reply to this email directly, view it on GitHub <https://github.com/ISET/iset3d/discussions/49> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABJ4N3GMTYRU3FLG3NAUKEDUW5DVDANCNFSM5MLI523A> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Comment by wandell** [comment]
I would never have asked you to do it.

But I wouldn’t stop you ;😉.

I think your big brain will be key to other lab stuff (more GPUs, mrMesh, next generation simulations).  Nobody is pounding down our door yet for documentation.  So, we can both do this in the background.

By the way – I think the first move to get stuff over is literally a copy and paste.  The hard part will be dealing with the changes in how we are doing the V4 lights and assets and materials.  So, we might wait until Zheng has done the lights, and then get him to write it.  Or Zhenyi has done the Blender integration and then get him to write it.

Brian

From: David Cardinal ***@***.***>
Date: Wednesday, January 19, 2022 at 5:15 PM
To: ISET/iset3d ***@***.***>
Cc: Brian A Wandell ***@***.***>, Author ***@***.***>
Subject: Re: [ISET/iset3d] Final version of V3 development (Discussion #49)
Because I’m not-smart-enough to know better, do you want me to start trying to move wiki pages to v4?





From: Brian Wandell ***@***.***>
Sent: Wednesday, January 19, 2022 3:40 PM
To: ISET/iset3d ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ISET/iset3d] Final version of V3 development (Discussion #49)



At this time, we are moving all of our new developments to ISET3d-V4. That repository may become the new 'ISET3d', and this one is likely to be renamed ISET3d-V3.

We will probably be copying the relevant portions of the wiki pages over in some way, as well.

Thank you PBRT V3. On to PBRT V4!

_____

This discussion was created from the release Final version of V3 development <https://github.com/ISET/iset3d/releases/tag/v3-final> .

—
Reply to this email directly, view it on GitHub <https://github.com/ISET/iset3d/discussions/49> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABJ4N3GMTYRU3FLG3NAUKEDUW5DVDANCNFSM5MLI523A> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Comment by digital-pro** [comment]
Brian – I’ve kind of “burnt the boats” or at least lit a match. I copied the v3 wiki over to the v4 wiki (which was basically empty anyway). I started to edit individual pages to bring them up to date. Obviously they’re all “not quite right” and need help, but I think this is a good way to get things started. I’ll go through and mark “v3” pages as such, so they can be updated. If instead you want to reset and go a page at a time, I’m okay with that also. -- David
